### PR TITLE
MONIT-31965 Add debug level payload logging

### PR DESCRIPTION
**Pull request by deosu**
Add a logger in LogDataSubmissionTask and log a payload before it is sent to the backend(if the root logger's level is defined as DEBUG).

### CODE REVIEW

**Review by xuranchen — APPROVED** [review]

File: proxy/src/main/java/com/wavefront/agent/data/LogDataSubmissionTask.java [hunk 1]
```diff
@@ -26,6 +28,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "__CLASS")
 public class LogDataSubmissionTask extends AbstractDataSubmissionTask<LogDataSubmissionTask> {
+  private static final Logger LOGGER = LogManager.getLogger(LogDataSubmissionTask.class);
```
Comment (laullon) [inline proxy/src/main/java/com/wavefront/agent/data/LogDataSubmissionTask.java:31]: Create a logger just for this.
https://github.com/wavefrontHQ/wavefront-proxy/blob/825e17e1c34f702656c2c9b7e1b429c313490a38/proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpTraceUtils.java#L167

Comment (deosu) [reply proxy/src/main/java/com/wavefront/agent/data/LogDataSubmissionTask.java:31]: @laullon Can you please explain a little bit? Are you suggesting creating a logger just for the `LogDataSubmissionTask.execute()`? Is the `OTLP_DATA_LOGGER` just an example? If yes, I have another question. The `LogDataSubmissionTask.java` class does not have any other functionality like the `OtlpTraceUtils.java` class. Does it make sense to create a logger just for the `execute()` method? Please guide.

Comment (deosu) [reply proxy/src/main/java/com/wavefront/agent/data/LogDataSubmissionTask.java]: @laullon please take a look again.

**Review by laullon — APPROVED** [review]

**Review by joannak-vmware — APPROVED** [review]
LGTM

**Pull request merged**